charts/umbrella/Chart.yaml

```
@@ -28,7 +28,7 @@ sources:
   - https://github.com/eclipse-tractusx/tractus-x-umbrella
 
 type: application
-version: 2.1.0
+version: 2.1.1
```

```suggestion
version: 2.2.0
```
The same as [here](https://github.com/eclipse-tractusx/tractus-x-umbrella/pull/195#discussion_r1923742316)